egs/wsj/s5/utils/validate_data_dir.sh

```diff
@@ -124,10 +124,10 @@ if [ ! -f $data/text ] && ! $no_text; then
 fi
 
 num_utts=`cat $tmpdir/utts | wc -l`
-if [ -f $data/text ]; then
+if ! $no_text; then
   if ! $non_print; then
     # compatible with bin/align-text
-    n_non_print=$(grep -c '[^[:print:][:space:]]' $data/text) && \
+    n_non_print=$(LC_ALL="C.UTF-8" grep -c '[^[:print:][:space:]]' $data/text) && \
```

Comment:
this isn't working for turkish. We should use en_us.UTF-8 here. Also I suggest we make non_print as true here, so that ppl only turn this on when necessary. Otherwise failing the program even if there's one offending line is too strict.

Reply:
cc @vimalmanohar

Reply:
This has already been fixed, see #4130

Reply:
hmm no.. Here we are using C.UTF-8, while it's still failing for turkish characters.. en_us.UTF-8 works

Reply:
Can you provide a test case?

Reply:
The correct setting here is `LANG=C.UTF-8`. grep does not care what is the first day of the week in the locale (if the C locale has them at all). either `LANG=` or `LC_CTYPE=` is enough.